source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -55,8 +61,13 @@ this was specific to the used setup.
 Screens
 ^^^^^^^
 
-Currently, Mixxx doesn't support controller screen rendering. However, because the screens are technically a different device, it is safe to interact with them outside of Mixxx while
-the device is being used by Mixxx. This can be particularly handy if you want to display some images or text, or even dynamic information.
+The controller screens are supported, but they require a little bit of tinkering. Due to the legacy controller engine, the screen are driven by a separate mapping, which will have to be set independently of the controller mapping.
+
+   .. note:: The screens rendering is done using QML and the communication is achieved via BULK USB endpoint. Both of these feature are not available out of the box on Windows and Mac for now.
+
+By default, the screens are standalone, meaning they do not interact with the device states. This means that local state such as deck selection or shifting will not trigger any screen interaction, and thus you should expect only Deck A and B to be displayed on these screen by default, although this can be changed in the controller settings.
+
+   .. note:: it exists an API that was temporary parked that allows mappings to communicate. If you want, you may build a version of Mixxx with feature and you should be able to enable the feature in the controller settings.
```

Comment:
Let's add a link to the branch/PR.
If someone is willing to try this it should be as easy as possible IMO.

Reply:
ah, it's mentioned at line 448